src/components/Header.jsx

```diff
@@ -16,7 +21,9 @@ function Header() {
 
   return (
     <div className={containerClassnames}>
-      <h1>Where in the world?</h1>
+      <h1 className={style.h1} onClick={handleClick}>
+        Where in the world?
+      </h1>
```

Comment:
I think we should use Link component here instead of handling navigation programatically.

Reply:
Nope, it's  not done. You just added a Link, but h1 still has onClick which does exactly the same as Link. You need to cleanup as well.